sdk/eventgrid/eventgrid-namespaces/src/eventGridNamespacesClient.ts

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureKeyCredential, TokenCredential } from "@azure/core-auth";
+import {
+  ReceiveResult,
+  AcknowledgeResult,
+  ReleaseResult,
+  RejectResult,
+  PublishCloudEventsOptions,
+  ReceiveCloudEventsOptions,
+  AcknowledgeCloudEventsOptions,
+  ReleaseCloudEventsOptions,
+  RejectCloudEventsOptions,
+  RenewCloudEventLocksOptions,
+  RenewCloudEventLocksResult,
+  CloudEvent,
+} from "./cadl-generated/models";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
use the uuid util function from core-util

Reply:
Done (in the next commit)